train/train_process.py

```diff
@@ -10,6 +10,7 @@
 from pathlib import Path
 import copy
 import json
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'

```

Comment:
change to 1? we still want to see TF warnings and errors

Reply:
done